wiring/src/spark_wiring_ble.cpp

```diff
@@ -2109,6 +2109,15 @@ int BleLocalDevice::setAdvertisingType(BleAdvertisingEventType type) const {
     return hal_ble_gap_set_advertising_parameters(&advParams, nullptr);
 }
 
+int BleLocalDevice::setAdvertisingPhy(EnumFlags<BlePhy> phy) const {
+    if(phy != BlePhy::CODED && phy != BlePhy::MBPS1) return SYSTEM_ERROR_NOT_SUPPORTED;
```

Comment:
We can remove this line, since we have the BLE HAL to check the validity already.

Reply:
Done.